AnisimovAN/Lab_5/Lab_5.cpp

```diff
@@ -0,0 +1,216 @@
+		}
+
+		cout << "Write elements of matrix A: \n";
+		for (i = 0; i < N; i++)
+		{
+			for (j = 0; j < M; j++)
+			{
+				cin >> mat_A[i][j];
+			}
+		}
+		cout << endl;
+
+		cout << "Matrix A \n";
+		for (i = 0; i < N; i++)
+		{
+			for (j = 0; j < M; j++)
+			{
+				cout << mat_A[i][j] << "\t";
+			}
+			cout << endl;
+		}
+		cout << endl;
+
+		cout << "Write a number of strings in matrix B (less then 21): ";
+		cin >> S;
+		cout << endl;
+
+		if (S > 20) {
+			do {
+				cout << "Error! Number of strings is too big, please write another one: " << endl;
+				cin >> S;
+			} while (S > 20);
+		}
+
+		cout << "Write a number of columns in matrix B (less then 21): ";
+		cin >> K;
+		cout << endl;
+
+		if (K > 20) {
+			do {
+				cout << "Error! Number of columns is too big, please write another one: " << endl;
+				cin >> K;
+			} while (K > 20);
+		}
+
+		cout << "Write elements of matrix B: \n";
+		for (i = 0; i < S; i++)
+		{
+			for (j = 0; j < K; j++)
+			{
+				cin >> mat_B[i][j];
+			}
+		}
+		cout << endl;
+
+		cout << "Matrix B \n";
+		for (i = 0; i < S; i++)
+		{
+			for (j = 0; j < K; j++)
+			{
+				cout << mat_B[i][j] << "\t";
+			}
+			cout << endl;
+		}
+		cout << endl;
+
+		cout << "Write a sign of operation (sum ' + ', dif ' - ', mult ' * ') : ";
+		cin >> sign;
+		cout << endl;
+
+		if (sign != '+' && sign != '-' && sign != '*') {
+			do {
+				cout << "Error! Wrong sign, please write another one: " << endl;
+				cin >> sign;
+			} while (sign != '+' && sign != '-' && sign != '*');
+		}
+		switch (sign)
+		{
+		case '+': if (N == S && M == K)
+		{
+			matrix_sum(*mat_A, *mat_B, *mat_C, N, M);
+			cout << "Matrix C \n";
+			for (i = 0; i < N; i++)
+			{
+				for (j = 0; j < M; j++)
+				{
+					cout << mat_C[i][j] << "\t";
+				}
+				cout << endl;
+			}
+		}
+				  else
+		{
+			cout << "Error: different number of strings or columns." << endl;
+		}
+
+				  break;
+
+		case '-': if (N == S && M == K)
+		{
+			matrix_difference(*mat_A, *mat_B, *mat_C, N, M);
+			cout << "Matrix C \n";
+			for (i = 0; i < N; i++)
+			{
+				for (j = 0; j < M; j++)
+				{
+					cout << mat_C[i][j] << "\t";
+				}
+				cout << endl;
+			}
+		}
+				  else
+		{
+			cout << "Error: different number of strings or columns." << endl;
+		}
+				  break;
+
+		case '*': if (N == K)
+		{
+			matrix_mult(*mat_A, *mat_B, *mat_C, N, M, S);
+			cout << "Matrix C \n";
+			for (i = 0; i < N; i++)
+			{
+				for (j = 0; j < M; j++)
+				{
+					cout << mat_C[i][j] << "\t";
+				}
+				cout << endl;
+			}
+		}
+				  else
+		{
+			cout << "Error: number of strings in 1'st Matrix is not the same as number of columns in 2'nd Matrix." << endl;
+		}
+
+				  break;
+		}
+		cout << endl;
+		cout << "--------------------------------END--------------------------------" << endl;
+		cout << endl;
+
+		cout << "Are you sure you want to quit this great program? (Press 'ESC' button to leave or another to continue)" << endl;
+
+		system("pause");
+		return 0;

```

Comment:
return call from main function will close the application, so user doesn't have any way to continue using it